webapp/packages/plugin-administration/src/ConfigurationWizard/ServerConfiguration/Form/ServerConfigurationInfoForm.tsx

```diff
@@ -74,6 +57,39 @@ export const ServerConfigurationInfoForm = observer<Props>(function ServerConfig
       >
         {translate('administration_configuration_wizard_configuration_server_session_lifetime')}
       </InputField>
+      <Switch
+        name="forceHttps"
+        state={state.serverConfig}
+        description={
+          <div className="tw:flex-col tw:gap-1 tw:flex tw:items-start">
+            <span>{translate('administration_configuration_wizard_configuration_secure_cookies_description')}</span>
+            <Link
```

Comment:
Probably we can use <Link indicator /> to simplify and make it consistence with other external links in the app?